main.go

```diff
@@ -71,10 +72,25 @@ func printLog(data BeaconLog) {
 	}
 }
 
+// loadConfig loads the .beaconrc file into the beaconConfig
+func loadConfig(config *beaconConfig) error {
+	configFile, err := ioutil.ReadFile("./.beaconrc")
+	if err != nil {
+		return err
+	}
+
+	return json.Unmarshal(configFile, &config)
+}
+
 // ============================
 // MAIN!
 // ============================
 func main() {
+	var config beaconConfig
+
+	if err := loadConfig(&config); err != nil {
+		log.Fatal(err)
```

Comment:
They are very similar. `panic` will print the stack trace. `log.Fatal` will prepend the datetime and call `os.Exit(1)` after printing the message but will not print the stack trace.

#### log.Fatal

```
2017/08/09 23:04:17 open ./.beacosnrc: no such file or directory
exit status 1
```

#### panic
```
panic: possibly missing .beaconrc file

goroutine 1 [running]:
main.loadConfig(0xc42000a400, 0xc42000a400, 0xc42001c0a0)
        /go/src/github.com/jbydeley/beacon/main.go:79 +0x111 main.main()
        /go/src/github.com/jbydeley/beacon/main.go:92 +0x5e
exit status 2
```